test/sampler/test_sampler.py

```diff
@@ -53,6 +59,8 @@
 hi = nk.hilbert.Spin(s=0.5, N=g.n_nodes)
 ha = nk.operator.Ising(hilbert=hi, graph=g, h=1.0)
 move_op = sum([nk.operator.spin.sigmax(hi, i) for i in range(hi.size)])
```

Comment:
can you try to add random weights in front of the sigma_x operators? this will allow to have a much more solid check of the sampler (for example, it would have caught the previous bug) !